e2e/driver/CheReporter.ts

```diff
@@ -79,17 +79,6 @@ class CheReporter extends mocha.reporters.Spec {
       }
     });
 
-    runner.on('test end', async function (test: mocha.Test) {
```

Comment:
This changes was implementing for 100% of the application state when test ends. Could you keep this changes? Of course it doesn't mean keeping of "10000" name.

Reply:
Could you, please, clarify for which  reasons we should keep this, taking into account that there is list of screenshots normally ordered now, and CheReporter make screenshot finally.

Reply:
Hehe ;-) I remember this conversation from the time you made this changes in the first place :-D